# Robot name goes here
supplybot:
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  
  F_Axle_Joint_Controller_R:
    type: effort_controllers/JointPositionController 
    joint: Axle_F_R_Joint  
    pid : {p: 100.0 , i: 0.0 , d: 0.0}

  F_Axle_Joint_Controller_L:
    type: effort_controllers/JointPositionController 
    joint: Axle_F_L_Joint  
    pid : {p: 100.0 , i: 0.0 , d: 0.0}

  R_Wheel_Joint_Controller_R:
    type: velocity_controllers/JointVelocityController 
    joint: Wheel_R_R_Joint  
    pid : {p: 100.0 , i: 0.0 , d: 0.0}

  R_Wheel_Joint_Controller_L:
    type: velocity_controllers/JointVelocityController 
    joint: Wheel_R_L_Joint  
    pid : {p: 100.0 , i: 0.0 , d: 0.0}
  
  Robot_Shoulder_Pan_Joint:
    type: effort_controllers/JointPositionController 
    joint: ur5_robot_shoulder_pan_joint  
    pid: {p: 1000.0, i: .001, d: 60.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  Robot_Shoulder_Lift_Joint:
    type: effort_controllers/JointPositionController 
    joint: ur5_robot_shoulder_lift_joint  
    pid: {p: 3000.0, i: 100.0, d: 175.0, i_clamp_min: -400.0, i_clamp_max: 400.0}
 
  Robot_Elbow_Joint:
    type: effort_controllers/JointPositionController 
    joint: ur5_robot_elbow_joint  
    pid: {p: 2500.0, i: 100.0, d: 100.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  Robot_Wrist_Joint_1:
    type: effort_controllers/JointPositionController 
    joint: ur5_robot_wrist_1_joint  
    pid: {p: 900.0, i: 10.0, d: 40.0, i_clamp_min: -400.0, i_clamp_max: 400.0}

  Robot_Wrist_Joint_2:
    type: effort_controllers/JointPositionController 
    joint: ur5_robot_wrist_2_joint  
    pid: {p: 900.0, i: 0.1, d: 10.0, i_clamp_min: -100.0, i_clamp_max: 100.0}

  Robot_Wrist_Joint_3:
    type: effort_controllers/JointPositionController 
    joint: ur5_robot_wrist_3_joint  
    pid: {p: 100.0, i: 0.1, d: 10.0, i_clamp_min: -100.0, i_clamp_max: 100.0}
